src/runtime/Instance.cpp

```diff
@@ -78,7 +98,7 @@ Memory* Instance::resolveExportMemory(std::string& name)
 {
     auto me = resolveExportType(name);
     if (me && me->exportType() == ExportType::Memory) {
-        return m_memories[me->itemIndex()];
+        return memory(me->itemIndex());
```

Comment:
Why did you change this part to call `memory` function instead of directly accessing `m_memories`?

Reply:
It was an early version of the code, it can be changed back.